Rakefile

```diff
@@ -0,0 +1,15 @@
+require 'asciidoctor/doctest'
+require 'thread_safe'
+require 'tilt'
+
+DocTest::RakeTasks.new do |t|
+  t.output_examples :html, path: 'test/output/slim'
+  t.input_examples :asciidoc, path: [ *DocTest.examples_path, 'examples' ]
+  t.converter = DocTest::HTML::Converter
+  t.converter_opts = { template_dirs: 'templates/slim' }
+end
+
+# TODO add a rake task to render fully working examples to look at them in a browser
```

Comment:
We should maybe create an issue to not forget to do it 😉

Reply:
Don't you always grep for `TODO|FIXME` before you do a release? ;)

Reply:
No, do you ? I grep from time to time but it's not integrated in my development process.
That's why I like to create issues on GitHub so I can keep track of what needs to be done.